ui/src/components/sortable/QasSortable.vue

```diff
@@ -109,9 +110,13 @@ export default {
       Loading.show()
 
       try {
-        const response = await this.$store.dispatch(`${this.entity}/replace`, {
-          payload: { order: this.identifiers },
-          url: this.url || `${this.entity}/sort`
+        const response = await getAction.call(this, {
+          entity: this.entity,
+          key: 'replace',
+          payload: {
+            payload: { order: this.identifiers },
```

Comment:
Será que seria legal renomear essa key `payload` interna lá no adapter? Para quando for pegar os valores desse objeto não ficar `payload.payload`...

Reply:
Acho que não, porque é para casos bem específicos, como precisa ser algo genérico, pode ser que acabe esbarrando em algo parecido